ex2_hal/adcs/hardware_interface/source/adcs.c

```diff
@@ -720,15 +720,15 @@ ADCS_returnState HAL_ADCS_get_track_controller(xyz *target) {
     #endif
 }
 
-ADCS_returnState HAL_ADCS_set_log_config(uint8_t flags_arr[10], uint16_t period, uint8_t dest, uint8_t log) {
+ADCS_returnState HAL_ADCS_set_log_config(uint8_t *flags_arr, uint16_t period, uint8_t dest, uint8_t log) {
```

Comment:
flags_arr[10] isn't really wrong, and it short of provides bounds checking. However, it is relatively unusual, so *flags_arr is fine.